plugins/civmodcore-paper/src/main/java/vg/civcraft/mc/civmodcore/events/EventUtils.java

```diff
@@ -1,11 +1,21 @@
 package vg.civcraft.mc.civmodcore.events;
 
 import java.util.Objects;
+import org.bukkit.event.Listener;
 import org.bukkit.event.player.PlayerInteractEvent;
 import org.bukkit.inventory.ItemStack;
+import org.bukkit.plugin.java.JavaPlugin;
 import org.jetbrains.annotations.NotNull;
 
 public final class EventUtils {
+    /** Convenience shortcut */
+    public static void registerListener(
```

Comment:
you don't need this

Reply:
It's not *needed*, sure, but it's a nice shortcut for civ plugins that don't extend `ACivMod`, like CastleGates, CombatTagPlus, ExilePearl, HiddenOre, RandomSpawn, and VoidWorld.

Reply:
It's not even convenient though

Reply:
That's because Java doesn't have extension methods D: